modules/src/clients/ics07_tendermint/client_state.rs

```diff
@@ -151,7 +151,7 @@ impl ClientState {
     }
 
     /// Helper method to produce a
-    /// [`tendermint_light_client::light_client::Options`] struct for use in
+    /// [`tendermint_light_client_verifier::options::Options`] struct for use in
     /// Tendermint-specific light client verification.
```

Comment:
It would be better to shorten the text in the link:

```rust
    /// [`Options`] struct for use in Tendermint-specific light client verification.
    /// [`Options`]: tendermint_light_client_verifier::options::Options
```

Reply:
Hmm I tried and it don't seem like this style of link shortening works in Rustdoc.